examples/xgboost_baseline.py

```diff
@@ -42,7 +50,13 @@
 dfs: Dict[str, pd.DataFrame] = {}
 entity_table = dataset.db.table_dict[task.entity_table]
 entity_df = entity_table.df
-col_to_stype = infer_df_stype(entity_df)
+
+if args.dataset in stype_dict:
+    col_to_stype = stype_dict[args.dataset][task.entity_table]
+else:
+    # sample 20_000 random rows to speed up infer_df_stype
+    entity_df = entity_df.sample(20_000, random_state=42)
```

Comment:
Nit: wouldn't this error out if `entity_df` has smaller than `20_000` rows, since `replace=False`?

Reply:
Removed this too - col types must now by hardcoded in stype.py for all datasets